front/tests/global-teardown.ts

```diff
@@ -2,24 +2,40 @@ import { test as setup } from '@playwright/test';
 
 import type { Infra, Project, RollingStock } from 'common/api/osrdEditoastApi';
 
-import { deleteApiRequest, getApiRequest } from './utils/index';
+import { deleteApiRequest, getApiRequest } from './utils/api-setup';
 
 setup('teardown', async () => {
-  const infras = await getApiRequest(`/api/infra/`);
-  const infra = infras.results.find((i: Infra) => i.name === 'small_infra_test_e2e');
+  // Fetch infra and projects in parallel
+  const [infras, projects] = await Promise.all([
+    getApiRequest('/api/infra/'),
+    getApiRequest('/api/projects/'),
+  ]);
 
-  const projects = await getApiRequest(`/api/projects/`);
+  const infra = infras.results.find((i: Infra) => i.name === 'small_infra_test_e2e');
   const project = projects.results.find((p: Project) => p.name === 'project_test_e2e');
 
-  const rollingStocks = await getApiRequest(`/api/light_rolling_stock/`, { page_size: 500 });
-  const electricRollingStock: RollingStock = await rollingStocks.results.find(
-    (r: RollingStock) => r.name === 'rollingstock_1500_25000_test_e2e'
-  );
-  const dualModeRollingStock: RollingStock = await rollingStocks.results.find(
-    (r: RollingStock) => r.name === 'dual-mode_rollingstock_test_e2e'
-  );
-  await deleteApiRequest(`/api/infra/${infra.id}/`);
-  await deleteApiRequest(`/api/projects/${project.id}/`);
-  await deleteApiRequest(`/api/rolling_stock/${electricRollingStock.id}/`);
-  await deleteApiRequest(`/api/rolling_stock/${dualModeRollingStock.id}/`);
+  // Fetch rolling stocks
+  const rollingStocks = await getApiRequest('/api/light_rolling_stock/', { page_size: 500 });
```

Comment:
Can you explain me why we use 500 here ?

Reply:
The default value of `page_size` is 25, which prevents listing all the rolling stock on the same page and makes parsing more difficult.